### DISCUSSION TITLE
How should I use "allowed domain" in create New Project page and how does it work?

### DISCUSSION THREAD

**Discussion by jaripy** [WalletConnect/walletconnect-monorepo#1728] [Developer Support]
We are planning to create our own wallet and we want to limit the range of DApps we can access.
Then, we planned to use `allowed domains` on the create New Project screen(image below) for the wallet.
May I please know how this option works?
<img width="487" alt="image" src="https://user-images.githubusercontent.com/69960640/206136812-ffda394c-9d30-4aad-b90b-f1786e0da4e3.png">

For example, in this image, if I want to only allow a Metamask DApps, shoud I set up like `metamask.io` ?
Alse, what happens if I do not set anything in this field? 

Thank you for your cooperation.

**Comment by chadyj** [comment]
The allowlist is to ensure your projectid credentials are only used on the domain you specify.

For example, if you created `app.example.com`, you would enter `app.example.com`.
This prevents `fakeapp.domain.com` from using your projectid credentials.

If you do not set anything, then you allow access from all domains.

**Reply by jaripy** [reply]
@chadyj 
Thank you for your support.
I understand the use of allowlist.

We would like to limit the number of DApps that can be linked from our wallet through WalletConnect to improve security, is there a better way?
If you have any ideas, I would glad to hear them.

**Reply by jaripy** [reply]
@chadyj 
Sorry for the additional question.
Do we need to create our own whitelist or something?

Thank you for your cooperation.

**Comment by jaripy** [comment]
Is it possible to use projectID which I recognize as being included in the url to carefully select the DApps to connect to?
